src/02-common-principles.md

```diff
@@ -544,6 +544,8 @@ For additional rules, see below:
 
 -   Frequency SHOULD be expressed in Hertz.
 
+-   Arbitrary/unitless units SHOULD be represented with the format `arbitrary`.
```

Comment:
"unitless units" feels odd. What about:

```suggestion
-   Arbitrary units SHOULD be indicated with the string `"arbitrary"`.
```

Reply:
That sounds fine to me, although for the folks who were originally using "unitless" (e.g., in BEP009), I'd like to make sure that it's clear that they'd use "arbitrary" in that case as well. @melanieganz WDYT?